applications/dashboard/scss/editor/_richEditor.scss

```diff
@@ -148,11 +152,11 @@ $embedded-quote_fg: #A3A3A3 !default;
 }
 
 // button reset
+.atMentionList-suggestion,
```

Comment:
Wouldn't it be better to give the suggestion a richEditor-button class, then than to add to the list here?

Reply:
`.richEditor-button` gets other styles that we don't want, like a set width and height since it expects to only contain an SVG.